app/controllers/sciruby_github_controller.rb

```diff
@@ -8,27 +8,11 @@ def repos
     # An example: http://nbviewer.jupyter.org/github/athityakumar/daru-io/blob/iruby-examples/iruby/json_importer.ipynb
     @df_repos = Daru::DataFrame.from_json('https://api.github.com/orgs/Sciruby/repos', Repo_name: '$..name', forks: '$..forks', open_issues: '$..open_issues_count')
 
-    opts = {
-      type: :column,
-      adapter: :googlecharts,
-      height: 500,
-      width: 1000
-    }
-    table_opts = {
-      adapter: :googlecharts, pageSize: 10,
-      height: 300, width: 400
-    }
-    @df_repo_table = Daru::View::Table.new(@df_repos, table_opts)
-    @df_repos_col_chart = Daru::View::Plot.new(@df_repo_table, opts)
-    opts = {
-      type: :histogram,
-      adapter: :googlecharts,
-      height: 500,
-      width: 1000
-    }
-    @df_repos_histo_chart = Daru::View::Plot.new(@df_repo_table, opts)
+    respond_to do |format|
+      format.html
 
-    export(@df_repos)
+      Daru::View::Helpers.respond(format, @df_repos, formats: %w[csv xls json])
```

Comment:
I am not sure about it (especially syntax), but I feel like task "this controller responds to those types, and returns exported dataframe" should be one-liner somehow, solved by daru-view, not by library user.